pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -204,7 +215,15 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 			"This will be a fatal error by default starting in K8s v1.6! " +
 			"In the meantime, you can opt-in to making this a fatal error by enabling --experimental-fail-swap-on.")
 	}
+	var capacity = v1.ResourceList{}
+	if info, err := cadvisorInterface.MachineInfo(); err == nil {
+		capacity = cadvisor.CapacityFromMachineInfo(info)
+	} else {
+		return nil, err
+	}
 
+	cgroupRoot := nodeConfig.CgroupRoot
```

Comment:
this is fine in a follow-on, but I think in future support issues, it would be a great help for us to have a log function that outputs what the node config was that we were given at this point.